client/components/domains/register-domain-step/index.jsx

```diff
@@ -923,9 +926,10 @@ class RegisterDomainStep extends React.Component {
 	getSubdomainSuggestions = ( domain, timestamp ) => {
 		const subdomainQuery = {
 			query: domain,
-			quantity: 1,
-			include_wordpressdotcom: ! this.props.includeDotBlogSubdomain,
+			quantity: this.props.includeDotBlogSubdomain ? 2 : 1,
```

Comment:
It makes more sense for this to live on the backend?

Reply:
I'd say no - Mobile app actually _does_ send a value much bigger than 1 or 2 (since they don't show any custom, paid domains).